### ISSUE TITLE
Validation is telling me that everything is ok but I'm still getting "We are missing some settings (...)" error

### ISSUE THREAD

**Issue by Pawel-Dymek** [mschmidtkorth/alfred-clickup-msk#17]
**Describe the bug**
`cu:config validate` is telling me that everything is ok but I'm still getting "We are missing some settings for ClickUp" error.
Strangely I can still search for tasks with `cus <search terms>`, but when I'm trying to use `cu <Title>` I'm getting said error.

One thing I can think of is that my Space ID does not look exactly like in examples.
Mine is `app.clickup.com/<Workspace ID>/v/l/4-2615579-1`

**Alfred version**
4.0.9

Thanks for any help in advance <3

**Comment by mschmidtkorth** [comment]
Hi @Pawel-Dymek thanks a lot for the report - I will look into it. Any chance you could provide the log files from Alfred? 

You can enable them at: Alfred Preferences > Workflows > ClickUp. On the top right, click on the little bug (rightmost icon) Then, try to use "cu <task>" again and copy the logs in the Alfred Preferences window.

**Comment by RvDelft** [comment]
If you can search in your ClickUp I think it is not an issue with the workspace ID. 
Did you write down the correct space and list ID? The List ID is between the two dashes. So in your URL example: 2615579

**Comment by Pawel-Dymek** [comment]
> Any chance you could provide the log files from Alfred?

Here you go
```[12:14:39.151] Logging Started...
[12:14:49.145] ClickUp[Script Filter] Queuing argument '(null)'
[12:14:49.371] ClickUp[Script Filter] Script with argv '(null)' finished
[12:14:49.374] STDERR: ClickUp[Script Filter] .
12:14:49 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:49 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:49 workflow.py:2254 DEBUG    set last run version: 1.0.7
12:14:49 workflow.py:2103 DEBUG    ---------- finished in 0.046s ----------
[12:14:49.378] ClickUp[Script Filter] {"items": [{"autocomplete": "apiKey ", "valid": false, "subtitle": "Your personal ClickUp API key/token.", "title": "Set API key (pk_4549558_37A3RAKPJS0F1GYPODJI5U1JZKXQBZMF)"}, {"autocomplete": "dueDate ", "valid": false, "subtitle": "e.g. m30 (in 30 minutes), h2 (in two hours), d1 (in one day), w1 (in one week)", "title": "Set default due date"}, {"autocomplete": "workspace ", "valid": false, "subtitle": "Workspace that defines which tasks can be searched", "title": "Set ClickUp workspace (2499279)"}, {"autocomplete": "space ", "valid": false, "subtitle": "Space that defines your available labels and priorities", "title": "Set ClickUp space (2605569)"}, {"autocomplete": "folder ", "valid": false, "subtitle": "Folder that which tasks can be searched. The Folder must be part of the workspace.", "title": "Set ClickUp folder"}, {"autocomplete": "list ", "valid": false, "subtitle": "List you want to add tasks to by default", "title": "Set default ClickUp list (17184943)"}, {"autocomplete": "notification ", "valid": false, "subtitle": "Show notification after creating task?", "title": "Set Show Notification (\u2713)"}, {"autocomplete": "defaultTag ", "valid": false, "subtitle": "Tag that is added to all new tasks.", "title": "Set default Tag (ClickUp)"}, {"autocomplete": "hierarchyLimit ", "valid": false, "subtitle": "Levels to lmit search results by (list, folder, space).", "title": "Set hierarchy levels to limit search results"}, {"autocomplete": "validate", "valid": false, "subtitle": "Check if provided configuration parameters are valid.", "icon": {"path": "./settings.png"}, "title": "Validate Configuration"}, {"subtitle": "Clear list of available labels and lists to be retrieved again.", "title": "Clear Cache", "variables": {"isSubmitted": "true"}, "valid": true, "arg": "cu:config cache", "icon": {"path": "./settings.png"}}]}
[12:14:50.580] ClickUp[Script Filter] Queuing argument ''
[12:14:50.795] ClickUp[Script Filter] Script with argv '' finished
[12:14:50.802] STDERR: ClickUp[Script Filter] .
12:14:50 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:50 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:50 workflow.py:2343 DEBUG    update check not due
12:14:50 main.py:597 DEBUG    [ main() ] - 
12:14:50 main.py:220 DEBUG    [ firstRun() ]
12:14:50 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:50 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:50 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:50 main.py:235 DEBUG    [ checkConfig() ]
12:14:50 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:50 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:50 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:50 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:50 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:50 main.py:237 DEBUG    Missing essential variables
12:14:50 workflow.py:2103 DEBUG    ---------- finished in 0.031s ----------
[12:14:50.804] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:51.694] ClickUp[Script Filter] Queuing argument 't'
[12:14:51.801] ClickUp[Script Filter] Queuing argument 'ta'
[12:14:51.925] ClickUp[Script Filter] Script with argv 't' finished
[12:14:51.933] STDERR: ClickUp[Script Filter] .
12:14:51 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:51 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:51 workflow.py:2343 DEBUG    update check not due
12:14:51 main.py:597 DEBUG    [ main() ] - t
12:14:51 main.py:220 DEBUG    [ firstRun() ]
12:14:51 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:51 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:51 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:51 main.py:235 DEBUG    [ checkConfig() ]
12:14:51 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:51 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:51 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:51 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:51 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:51 main.py:237 DEBUG    Missing essential variables
12:14:51 workflow.py:2103 DEBUG    ---------- finished in 0.043s ----------
[12:14:51.936] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:52.028] ClickUp[Script Filter] Queuing argument 'tas'
[12:14:52.136] ClickUp[Script Filter] Script with argv 'ta' finished
[12:14:52.140] STDERR: ClickUp[Script Filter] .
12:14:52 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:2343 DEBUG    update check not due
12:14:52 main.py:597 DEBUG    [ main() ] - ta
12:14:52 main.py:220 DEBUG    [ firstRun() ]
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:52 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:52 main.py:235 DEBUG    [ checkConfig() ]
12:14:52 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 main.py:237 DEBUG    Missing essential variables
12:14:52 workflow.py:2103 DEBUG    ---------- finished in 0.038s ----------
[12:14:52.143] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:52.321] ClickUp[Script Filter] Queuing argument 'task'
[12:14:52.331] ClickUp[Script Filter] Script with argv 'tas' finished
[12:14:52.336] STDERR: ClickUp[Script Filter] .
12:14:52 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:2343 DEBUG    update check not due
12:14:52 main.py:597 DEBUG    [ main() ] - tas
12:14:52 main.py:220 DEBUG    [ firstRun() ]
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:52 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:52 main.py:235 DEBUG    [ checkConfig() ]
12:14:52 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 main.py:237 DEBUG    Missing essential variables
12:14:52 workflow.py:2103 DEBUG    ---------- finished in 0.031s ----------
[12:14:52.339] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:52.544] ClickUp[Script Filter] Script with argv 'task' finished
[12:14:52.554] STDERR: ClickUp[Script Filter] .
12:14:52 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:2343 DEBUG    update check not due
12:14:52 main.py:597 DEBUG    [ main() ] - task
12:14:52 main.py:220 DEBUG    [ firstRun() ]
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:52 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:52 main.py:235 DEBUG    [ checkConfig() ]
12:14:52 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:52 main.py:237 DEBUG    Missing essential variables
12:14:52 workflow.py:2103 DEBUG    ---------- finished in 0.030s ----------
[12:14:52.556] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:53.149] ClickUp[Script Filter] Queuing argument 'task '
[12:14:53.333] ClickUp[Script Filter] Queuing argument 'task n'
[12:14:53.354] ClickUp[Script Filter] Script with argv 'task ' finished
[12:14:53.357] STDERR: ClickUp[Script Filter] .
12:14:53 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2343 DEBUG    update check not due
12:14:53 main.py:597 DEBUG    [ main() ] - task 
12:14:53 main.py:220 DEBUG    [ firstRun() ]
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 main.py:235 DEBUG    [ checkConfig() ]
12:14:53 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 main.py:237 DEBUG    Missing essential variables
12:14:53 workflow.py:2103 DEBUG    ---------- finished in 0.032s ----------
[12:14:53.359] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:53.409] ClickUp[Script Filter] Queuing argument 'task na'
[12:14:53.516] ClickUp[Script Filter] Queuing argument 'task nam'
[12:14:53.576] ClickUp[Script Filter] Script with argv 'task n' finished
[12:14:53.580] STDERR: ClickUp[Script Filter] .
12:14:53 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2343 DEBUG    update check not due
12:14:53 main.py:597 DEBUG    [ main() ] - task n
12:14:53 main.py:220 DEBUG    [ firstRun() ]
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 main.py:235 DEBUG    [ checkConfig() ]
12:14:53 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 main.py:237 DEBUG    Missing essential variables
12:14:53 workflow.py:2103 DEBUG    ---------- finished in 0.031s ----------
[12:14:53.582] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:53.595] ClickUp[Script Filter] Queuing argument 'task name'
[12:14:53.784] ClickUp[Script Filter] Script with argv 'task nam' finished
[12:14:53.791] STDERR: ClickUp[Script Filter] .
12:14:53 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2343 DEBUG    update check not due
12:14:53 main.py:597 DEBUG    [ main() ] - task nam
12:14:53 main.py:220 DEBUG    [ firstRun() ]
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 main.py:235 DEBUG    [ checkConfig() ]
12:14:53 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 main.py:237 DEBUG    Missing essential variables
12:14:53 workflow.py:2103 DEBUG    ---------- finished in 0.032s ----------
[12:14:53.794] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}
[12:14:53.984] ClickUp[Script Filter] Script with argv 'task name' finished
[12:14:53.990] STDERR: ClickUp[Script Filter] .
12:14:53 workflow.py:2061 DEBUG    ---------- ClickUp (1.0.7) ----------
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2343 DEBUG    update check not due
12:14:53 main.py:597 DEBUG    [ main() ] - task name
12:14:53 main.py:220 DEBUG    [ firstRun() ]
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 workflow.py:2225 DEBUG    last run version: 1.0.7
12:14:53 main.py:235 DEBUG    [ checkConfig() ]
12:14:53 workflow.py:2461 DEBUG    got password : com.msk.clickup2.0:clickUpAPI
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 workflow.py:1468 DEBUG    reading settings from /Users/paweldymek/Library/Application Support/Alfred/Workflow Data/com.msk.clickup2.0/settings.json
12:14:53 main.py:237 DEBUG    Missing essential variables
12:14:53 workflow.py:2103 DEBUG    ---------- finished in 0.031s ----------
[12:14:53.993] ClickUp[Script Filter] {"items": [{"arg": "cu:config ", "valid": true, "subtitle": "Let's set it up?", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "We are missing some settings for ClickUp."}]}```

**Comment by Pawel-Dymek** [comment]
> If you can search in your ClickUp I think it is not an issue with the workspace ID.
> Did you write down the correct space and list ID? The List ID is between the two dashes. So in your URL example: 2615579

@RvDelft never can be too sure, but I've checked it many times

**Comment by mschmidtkorth** [comment]
Hi @Pawel-Dymek did you specify a Folder Id? It used to be required but is now optional, though there was still a check as if it were required, displaying the 'We are missing...' error message if you haven't specified one. I've corrected that and will submit a new build once you confirm, because that is very likely to be the cause. Otherwise I'll investigate further.

**Comment by Pawel-Dymek** [comment]
Hi @mschmidtkorth :) 
No, I have not specified any folders, because my preferred location is a list living in a space with no folders

**Comment by mschmidtkorth** [comment]
@Pawel-Dymek This should be fixed with the newest version [0.1.0](https://github.com/mschmidtkorth/alfred-clickup-msk/releases/tag/1.1.0). This version also adds a new command, `cuo` to show open tasks.
